OrbitGl/TimeGraph.cpp

```diff
@@ -377,6 +383,88 @@ void TimeGraph::ProcessTimer(const TimerInfo& timer_info) {
   NeedsUpdate();
 }
 
+//-----------------------------------------------------------------------------
+void TimeGraph::ProcessOrbitFunctionTimer(const FunctionInfo* function,
+                                          const TimerInfo& timer_info) {
+  FunctionInfo::OrbitType type = function->type();
+  uint64_t time = timer_info.start();
+  CHECK(timer_info.registers_size() > 0);
+  uint64_t graph_id = timer_info.registers(0);
+
+  switch (type) {
+    case FunctionInfo::kOrbitTrackInt: {
+      int32_t value = static_cast<int32_t>(timer_info.registers(1));
```

Comment:
This is why we need an explicit mapping of registers. What registers is this? Why 1? Either way, at least use a constant in place of `1` so that we give it a name.